tests/expose_test.go

```diff
@@ -407,6 +407,67 @@ var _ = Describe("Expose", func() {
 
 				Eventually(getStatus, 60, 1).Should(Equal(k8sv1.PodSucceeded))
 			})
+
+			It("[test_id:345]Should verify the exposed service is functional before and after VM restart.", func() {
+				vmObj := vm
+
+				By("Getting back the service's allocated cluster IP.")
+				svc, err := virtClient.CoreV1().Services(vmObj.Namespace).Get(serviceName, k8smetav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+				serviceIP := svc.Spec.ClusterIP
+
+				By("Starting a pod which tries to reach the VMI via ClusterIP.")
+				job := tests.NewHelloWorldJob(serviceIP, servicePort)
+				job, err = virtClient.CoreV1().Pods(vmObj.Namespace).Create(job)
+				Expect(err).ToNot(HaveOccurred())
+
+				By("Waiting for the pod to report a successful connection attempt.")
+				getStatus := func() k8sv1.PodPhase {
```

Comment:
Can you please create a  function for this we call it 3 times in the code

something like 
```golang
func getStatus(job k8sv1.Job) {
pod, err := virtClient.CoreV1().Pods(job.Namespace).Get(job.Name, k8smetav1.GetOptions{})
					Expect(err).ToNot(HaveOccurred())
					return pod.Status.Phase
				}

 				Eventually(getStatus, 60, 1).Should(Equal(k8sv1.PodSucceeded))
}
```

Reply:
@SchSeba I already merged this PR.
Good point, though. I might implement it on another PR.